tests/test_main.py

```diff
@@ -350,6 +350,11 @@ def test_packages_distributions_all_module_types(self):
         distributions = packages_distributions()
 
         for i in range(len(suffixes)):
-            assert distributions[f'{i}-top-level'] == ['all_distributions']
-            assert distributions[f'{i}-in-namespace'] == ['all_distributions']
-            assert distributions[f'{i}-in-package'] == ['all_distributions']
+            assert distributions[f'top_level_{i}'] == ['all_distributions']
+            assert distributions[f'in_namespace_{i}'] == ['all_distributions']
+            assert distributions[f'in_package_{i}'] == ['all_distributions']
+
+        # All keys return from packages_distributions() should be valid
+        # import names, which means that they must _at least_ be valid
+        # identifiers:
+        assert all(import_name.isidentifier() for import_name in distributions.keys())
```

Comment:
This assertion is incorrect. Module names are allowed not to be identifiers when invoked by runpy. For example, `py -m pip-run` is a valid invocation of a file called `pip-run.py`. We'll need to find a way to more precisely describe the expected inclusions/exclusions.

Reply:
Addressed in 7968a08.